### PR TITLE
Document auto-publish feature

### PR DESCRIPTION
**Pull request by dralley**
closes: #8548
https://pulp.plan.io/issues/8548

### CODE REVIEW

File: docs/workflows/publish-host.rst [hunk 1]
```diff
@@ -55,3 +56,18 @@ If you created your repository version using the :doc:`upload` workflow:
...
+Automate Publication and Distribution
+-------------------------------------
+
+With a little more initial setup, you can have publications and distributions for your repositories
+updated automatically when new repository versions are created.
+
+.. code-block:: bash
+
+    # This configures the repository to produce new publications when a new version is created
+    pulp file repository update --name $REPO_NAME --autopublish True
```
Comment (daviddavis) [inline docs/workflows/publish-host.rst:69]:
```suggestion
    pulp file repository update --name $REPO_NAME --autopublish
```

We're generally using flags (eg `--auto-publish/--no-autopublish`) to represent booleans in the cli.

**Comment by daviddavis** [comment]
Can you add a warning about this feature being in tech preview?

**Comment by dralley** [comment]
@daviddavis Yup, but I probably won't touch this PR until the CLI PR is merged.  This is just what I had from the other day.

**Comment by daviddavis** [comment]
Here's the CLI PR if you want to review or test it out: https://github.com/pulp/pulp-cli/pull/199

**Review by daviddavis — APPROVED** [review]

**Pull request merged**